### DISCUSSION TITLE
Ridurre tempo di latenza IKEA motion sensor

### DISCUSSION THREAD

**Discussion by sgravagnola** [indomus/forum#394] [HUB: Home Assistant]
Salve a tutti,
ho preso il sensore di movimento di IKEA e l'ho collegato alla mia chiavetta per zigbee2mqtt.

Ho poi realizzato una banale automazione per accendere la luce al passaggio e tutto funziona correttamente.

Il problema però è che il sensore, una volta attivato, torna nello stato off dopo 3 minuti (salvo ulteriori movimenti in questo lasso di tempo).

E' possibile in qualche modo ridurre questo tempo? Sono a conoscenza del fatto che questo ridurrebbe la durata della batteria, ma vorrei provare a fare qualche prova per trovare il giusto equilibrio tra latenza necessaria a me e durata della batteria.

Qualcuno può aiutarmi?

Grazie in anticipo.

**Comment by gforment** [comment]
Se vai nella documentazione zigbee2mqtt del device che usi trovi il parametro "occupacy_timeout" da settare in configurazione. Prova a leggere le indicazioni per evitare problemi :)

**Comment by sgravagnola** [comment]
Grazie, per caso sai dirmi come fare la stessa operazione utilizzando l'integrazione Zigbee Home Automation (ZHA)?

**Reply by gforment** [reply]
Ciao, ho abbandonato ZHA perché mi sembrava fra le altre cose che garantisse meno configurabilità come in questo caso... io non ero riuscito... 😊

**Reply by snakuzzo** [reply]
Come ti diceva @gforment 

https://www.zigbee2mqtt.io/devices/E1525_E1745.html#detection-cooldown-e1745-version

Su ZHA non puoi fare nulla...dovresti provare a gestirtela lato HA, ma non saprei come sinceramente
